presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1179,11 +1179,9 @@ The connector maps Iceberg types to the corresponding PrestoDB types:
   * - ``STRING``
     - ``VARCHAR``
   * - ``LIST``
-    - ``LIST``
+    - ``ARRAY``
   * - ``MAP``
     - ``MAP``
-  * - ``STRUCT``
-    - ``ROW``
```

Comment:
Why remove iceberg ``STRUCT`` to presto ``ROW`` mapping?

Reply:
Please see comment above: https://github.com/prestodb/presto/pull/21998#discussion_r1509248887. I will add it back.